tests/network-tests/package.json

```diff
@@ -7,7 +7,7 @@
     "test": "tap --files ts-node/register src/nicaea/tests/proposals/*Test.ts --files ts-node/register src/nicaea/tests/workingGroup/*Test.ts -T",
     "test-migration-constantinople": "tap --files src/rome/tests/romeRuntimeUpgradeTest.ts --files src/constantinople/tests/electingCouncilTest.ts -T",
     "test-migration-nicaea": "tap --files src/constantinople/tests/proposals/updateRuntimeTest.ts --files src/nicaea/tests/electingCouncilTest.ts -T",
-    "debug": "tap --files src/nicaea/tests/proposals/manageLeaderRole.ts -T",
+    "debug": "tap --files ts-node/register src/nicaea/tests/workingGroup/*Test.ts -T",
```

Comment:
what is `ts-node/register` ?

Reply:
artifact left from attempt to run the tests with node instead of tap. Removed now.